packages/utilities/src/Async.ts

```diff
@@ -141,6 +143,7 @@ export class Async {
    */
   public setImmediate(callback: () => void): number {
     let immediateId = 0;
+    const win = getWindow()!;
```

Comment:
Do you want to allow a passthrough for an htmlelement or react component to this?